src/calendar-to-sheets/tests/index.test.js

```
@@ -1119,6 +1119,169 @@ describe('Checkpoint logic (GAS only)', () => {
     delete global.SYNC_CONFIGS
   })
 
+  test('fullResyncCalendarToSheetGAS deletes rows for a specific config', () => {
+    const code = require('../code.gs');
```

ESLint

[error] Delete `;`

(prettier/prettier)